query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/DeleteMutationRelationsSpec.scala

```diff
@@ -7,103 +7,6 @@ import util._
 class DeleteMutationRelationsSpec extends FlatSpec with Matchers with ApiSpecBase with SchemaBaseV11 {
   override def runOnlyForCapabilities: Set[ConnectorCapability] = Set(JoinRelationLinksCapability)
 
-  "a P1! to C1! relation " should "error when deleting the parent" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.